daml-lf/encoder/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ lf_targets = [
     "1.3",
     "1.6",
     "1.7",
+    "1.8",
```

Comment:
This require also update of file names in  `daml-lf/encoder/src/test/lf` 
(I know this is ugly)

Reply:
I also expect a test for synonyms here.

Reply:
I’ve updated this and added a test for type synonyms which works (after actually implementing the encoder).

Reply:
:+1: